From fc12c98e8cb2ac923fb1c7cffd19403f5706833d Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 1 Mar 2018 12:39:48 -0800
Subject: [PATCH] TRITON-200 sdc-docker cli-nfs-shared-volumes-disabled.test.js
 is flaky Reviewed by: Josh Wilsdon <josh@wilsdon.ca> Approved by: Josh
 Wilsdon <josh@wilsdon.ca>

---
 test/integration/cli-nfs-shared-volumes-disabled.test.js | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/integration/cli-nfs-shared-volumes-disabled.test.js b/test/integration/cli-nfs-shared-volumes-disabled.test.js
index 71df3b6..7424e97 100644
--- a/test/integration/cli-nfs-shared-volumes-disabled.test.js
+++ b/test/integration/cli-nfs-shared-volumes-disabled.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -34,7 +34,7 @@ var NFS_SHARED_VOLUME_NAMES_PREFIX =
     testVolumes.getNfsSharedVolumesNamePrefix();
 var SAPI_CLIENT;
 var SAPI_APP;
-var STATE_RETRIES = 120;
+var STATE_RETRIES = 180;
 
 // wait for /admin/config to have state either 'enabled' or 'disabled' for
 // experimental_docker_nfs_shared_volumes
@@ -137,8 +137,8 @@ test('setup', function (tt) {
                         + app.metadata.experimental_docker_nfs_shared_volumes);
                 }
 
-                if (app.metadata.experimental_docker_nfs_shared_volumes ===
-                    false) {
+                if (app.metadata.experimental_docker_nfs_shared_volumes !==
+                    true) {
                     t.comment('NFS volumes support already disabled, no need '
                         + 'to turn it off');
                     t.end();
-- 
2.21.0

